lib/tiler.css

```diff
@@ -1,6 +1,7 @@
 .tiler-viewport {
-  overflow: hidden;
-  position: relative; }
+  position: relative;
+  background: white;
```

Comment:
i think these particular properties were already in the checked in src, just not the compiled lib files.... sloppy incomplete release basically...